src/pages/workspaces/workspace/Dashboard/Dashboard.ts

```diff
@@ -114,11 +114,12 @@ const WorkspaceDashboardForwardRefRenderFunction = (
     workspace: {
       accessLevel,
       owners = [],
-      workspace: { authorizationDomain, attributes = { description: '' } },
+      workspace: { authorizationDomain, attributes },
     },
   } = props;
 
-  const description = attributes.description;
+  // attributes.description will be undefined for workspaces where it has never been set
+  const description = _.isString(attributes['description']) ? attributes['description'] : ''; // eslint-disable-line dot-notation
```

Comment:
In addition to the bug where one couldn't edit a workspace that had an undefined `description`, I was seeing an issue where I would hit edit on a workspace that _appeared_ to have no description, but then the editor would show that it actually did have a description (which would then display, even after refreshing the page). I believe that was intended to be a default value of '' at line 117 was causing the description to be initially overwritten with the empty string. Oddly though, this bug didn't seem to always reproduce (though I saw it twice).

It seems safer to just handle `description` in the same way that we do with other attributes.